src/userstyles.yml

```diff
@@ -272,13 +272,13 @@ userstyles:
     readme:
       app-link: "https://github.com/mastodon/mastodon"
       usage: |+
-        Add urls to `@-moz-document domain("url")` with url being the chosen server
+        Add URLs to `@-moz-document domain("url")` with url being the chosen server.
```

Comment:
This seems poorly phrased but I don't know enough about `@-moz-document domain()` to fix it. Maybe something like "Add Mastodon server URLs to `@-moz-document domain("url")`, e.g. `@-moz-document domain(some example here...)`."

Reply:
I agree what you mean in terms of phrasing but im unsure what would be better strictly speaking and currently it makes enough sense to get by.
In terms of examples i don't think its needed since the actual user.css contains some examples in context that are a strong hint enough for the user

Reply:
What if we use the examples from the `user.css` in the README?

> Add your Mastodon server URLs to `@-moz-document domain("url")`, like this:
> ```
> @-moz-document domain("mstdn.social"),
> domain("fosstodon.org"),
> domain("tech.lgbt"),
> domain("mastodon.social")
> ```

Reply:
Update on this @isabelroses?

Reply:
I mean that could make sense but only 2 or so would be more than enough to show how it is done.

Reply:
How about this?

Add your Mastodon server URLs at the top of the userstyle, like this:
```
@-moz-document domain("mastodon.social"), domain("fosstodon.org") {
```

Reply:
Sounds good to me

Reply:
Or maybe we change this part of the `user.css` to this:

```diff
- @-moz-document domain("mstdn.social"),
- domain("fosstodon.org"),
- domain("tech.lgbt"),
- domain("mastodon.social"),
- domain("piaille.fr") {
+  @-moz-document domain("mastodon.social") {
```

And have the README say:

Replace `@-moz-document domain("mastodon.social") {` at the top of the userstyle with the Mastodon server URLs that you use, like this:

```css
@-moz-document domain("mastodon.social"), domain("fosstodon.org") {
```

Reply:
I think, this makes the user experience worse, i would prefer to have 100+ default urls then to lose any for the point of an example.
But we can certainly put a comment notice in to tell the use to do something similar for their own instance

Reply:
I think we should aim to be as user friendly as possible and make sure we are careful about how we guide people who may not be comfortable with using/editing userstyles. Saying to "add your Mastodon server URLs at the top of the userstyle" might be confusing about where exactly the should add it, if they can remove the other URLs there, etc. I also think comments in the userstyle are much less likely to be read than if you put a note in the README.

Also another thought just occurred to me - won't editing the URLs be removed when you update the userstyle? Should there be a note about that?

Reply:
Yeah, the searxng port has a similar issue where you can have multiple urls, they have all the default urls, and users can add their own instance if they wish.

What we are aiming for here is something similar.  Where a user will find it simple to add their own instances if they need, but ideally they would not have to due to how many instances we support by default.

Reply:
I'm thinking...

---

This userstyle is applied by default on a few common Mastodon server URLs, listed at the top:
```css
@-moz-document domain("mastodon.social"), domain("mstdn.social"), domain("fosstodon.org"), domain("tech.lgbt") {
```
If you don't see your Mastodon server URL, feel free to add it to the list with the `domain("<URL>")` syntax.

---

And what do you think about my other comment?
> won't editing the URLs be removed when you update the userstyle? Should there be a note about that?

Reply:
I love the new note suggestion i think it captures the situation perfectly.

> won't editing the URLs be removed when you update the userstyle? Should there be a note about that?

In terms of this yes it will. I think that:
- We can prompt users to either contribute their instance to the userstyle domain pool
- re add their instance upon update

Reply:
Alright. I'll just put the clarified note in the README and maybe you can add the two bullet points you suggested in a separate PR?

Reply:
Sure that makes sense